### PR TITLE
Armenia

### PR DESCRIPTION
**Pull request by Vesihiisi**
Mapping: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/am_(hy)
Preview: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/am_(hy)/preview

Task: https://phabricator.wikimedia.org/T176710

### CODE REVIEW

File: importer/AmHy.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        """
+        if "monuments" in self.commonscat.lower():
+            return
+        else:
+            super().set_commonscat()
+
+    def set_heritage_id(self):
+        self.add_statement("armenia_heritage", str(self.id))
+
+    def exists_with_monument_article(self, language):
```
Comment (lokal-profil) [inline importer/AmHy.py]: can be dropped

**Comment by lokal-profil** [comment]
A follow up on the `registrant_url` comment. Looks like there is an issue with the harvesting [T176991](https://phabricator.wikimedia.org/T176991) so it might have to be constructed manually here.

File: importer/AmHy.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    def update_labels(self):
+        name = utils.remove_markup(self.description)
+        self.add_label("hy", name)
+
+    def update_descriptions(self):
+        english = "cultural heritage monument of Armenia"
+        self.add_description("en", english)
+
+    def set_adm_location(self):
```
Comment (lokal-profil) [inline importer/AmHy.py]: So I just spotted that the list also includes monuments in Nagorno-Karabakh (officially Azerbajdzjan). We therefore need to include a data_file for those communities and possibly manually create a data file for those regions (they have non-ISO iso codes which start with NK).

Additionally we need to set country to Q227 (Azerbajdzjan) for these.

**Comment by Vesihiisi** [comment]
Added 'described at url' to heritage id and tested in sandbox.

File: importer/AmHy.py (hunk 1 above)
Comment (Vesihiisi) [reply importer/AmHy.py]: Done.

File: importer/AmHy.py (hunk 2 above)
Comment (Vesihiisi) [reply importer/AmHy.py]: Note: All the possible prov_iso are [here](https://gist.github.com/Vesihiisi/ffdbdcdd36abdcc012aa5704c8143399). So making a manual file for the NK ones

**Comment by Vesihiisi** [comment]
Preview up to here.

**Pull request merged**